certifikit/src/main/kotlin/app/cash/certifikit/certificates.kt

```diff
@@ -30,6 +30,12 @@ data class Certificate(
   val signatureAlgorithm: AlgorithmIdentifier,
   val signatureValue: BitString
 ) {
+  /**
+   * Certificate hash as used in HTTP Public Key Pinning.
+   */
+  fun sha256Hash(): ByteString =
```

Comment:
I think we should put pin in the name here. The hash of the cert covers more stuff (subject alt names, expiration, validity) and trusting a cert with one hash is not the same as trusting another one with the same hash.